modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -29,8 +29,116 @@ payload
+
+Valid values are `true` or `false`.
+|`useMissCharAsDefaultForFill`  |`Boolean`|`false`|By default, the flat file reader and writer use spaces for missing characters and ignore the setting of the `missingValues` property. When you set this property to `true`, DataWeave honors the setting of the `missingValues property`. Introduced in DataWeave 4.3 (2.3.0-20210823) for the September 2021 release of Mule 4.3.0-20210823.
+
+Valid values are `true` or `false`.
+|`zonedDecimalStrict`  |`Boolean`|`false`|For COBOL copybook, uses the 'strict' ASCII form of sign encoding for zoned decimal values.
+
+Valid values are `true` or `false`.
+|===
+
+[[writer_properties]]
+=== Writer Properties
+
+This format accepts properties that provide instructions for writing output data.
+
+[cols="1,1,1,3a", options="header"]
+|===
+|Parameter |Type |Default|Description
+|`bufferSize`  |`Number`|`8192`|Size of the buffer writer. The value must be greater than 8.
+
+|`deferred`  |`Boolean`|`false`|Generates the output as a data stream when set to `true`, and defers the script's execution until the generated content is consumed.
+
+
+Valid values are `true` or `false`.
+|`encoding`  |`String`|`null`|The encoding to use for the output, such as UTF-8.
+
+|`enforceRequires`  |`Boolean`|`false`|Produces an error when set to `true` if a required value is missing.
+
+Valid values are `true` or `false`.
+|`missingValues`  |`String`|`null`|Fill character used to represent missing values. To activate a non-default setting, set the `useMissCharAsDefaultForFill` property to `true`, and use one of the following values to `missingValues`:
+
+* `none` (for the reader) or `NONE` (for the writer): Treats all data as values.
+* `spaces` (for the reader) or `SPACES` (for the writer): Interprets a field consisting of only spaces as a missing value. Default for flat file and fixed-width formats.
+* `zeroes` (for the reader) or `ZEROES` (for the writer): Interprets numeric fields consisting of only `0` characters _and_ character fields consisting of only spaces as missing values.
+* `nulls` (for the reader) or `NULLS` (for the writer): Interprets a field consisting only of `0` bytes as a missing value. Default for COBOL copybook schema.
+
+Valid values are `none` or `spaces` or `zeroes` or `nulls`.
+|`notTruncateDependingOnSubjectNotPresent`  |`Boolean`|`false`|Fills the entire group when the DEPENDING ON subject is not present.
+
+Valid values are `true` or `false`.
+|`recordTerminator`  |`String`|`null`|Line break for a record separator.
+
+Valid values are `lf` or `cr` or `crlf` or `none`.
+|`schemaPath` (Required) |`String`|`null`|Path to the schema definition. Specifies the location in your local disk of the schema file that parses your input.
+|`segmentIdent`  |`String`|`null`|Segment identifier in the schema for fixed-width or COBOL copybook schemas. Required when parsing a single segment or record definition if the schema includes multiple segment definitions.
```

Comment:
`| `segmentIdent` | `String` | `null` | Segment identifier in the schema for fixed width or copybook schemas (only needed when writing a single segment/record definition, and if the schema includes multiple segment definitions).`

Reply:
looks fine